techlibs/xilinx/synth_xilinx.cc

```diff
@@ -78,6 +81,12 @@ struct SynthXilinxPass : public ScriptPass
 		log("    -nowidelut\n");
 		log("        do not use MUXF[78] resources to implement LUTs larger than LUT6s\n");
 		log("\n");
+		log("    -iopads\n");
```

Comment:
Perhaps we should have just one option instead of two: `-iopad` (non plural to be consistent) to make it clearer.
In the unlikely scenario that the user wishes to disable under `-ise`, perhaps they can do `-iopad=false`?

Reply:
Certainly could be done, but I see no other option in yosys that takes a bool like that, and there is a precedent for -<option> and -no<option> (read_verilog -[no]mem2reg, -[no]overwrite). Agreed about non plural.

Reply:
Just an idea to reduce the number of yes/no flags, but you're right, there's no precedent. Let's leave as-is.